### ISSUE TITLE
Empty lists are being exported

### ISSUE THREAD

**Issue by lzilioli** [wormi4ok/evernote2md#27]
**Problem statement**

I have a note which, when exported, is missing some data. The following zip archive illustrates the issue:

**Example file**

[Archive.zip](https://github.com/wormi4ok/evernote2md/files/5766580/Archive.zip)

You can open up the .enex and see that each list item has contents "list item" however the exported note contains nothing.

I ran the following command:

```
$ evernote2md -v Actual.enex && cat notes/Exported_List_is_Empty.md
2021/01/04 15:01:44 [DEBUG] Saving file ./notes/Exported_List_is_Empty.md
Done!
```

And the result:

```
$ cat notes/Exported_List_is_Empty.md
# Exported List is Empty

`Sample Tag`

Paragraph

*

*

*

*

*

*

*

*

*
Paragraph
*

*

*

*

*

*
Paragraph
*

*

*

*

*

    *

    *

    *

    *

    *

    *

    *

    *
Paragraph
ParagraphParagraph
*

*

*

*

*

*

*

*

*

*
Paragraph
*

*

*

*

*

*

*

*

*

*

*

*

*
Paragraph
*

*

*

*

*

*

*

*

*

*

*

*

*

*

*```

**Comment by wormi4ok** [comment]
Yes, @lzilioli, I can confirm the bug, and it was most likely introduced while working on #10. 

Thanks for reporting, I'll try to fix it in the next release.

**Comment by wormi4ok** [comment]
Fixed in version [0.14.0](https://github.com/wormi4ok/evernote2md/releases/tag/v0.14.0)
